watcher/watcher.go

```diff
@@ -282,6 +282,18 @@ func (wh *WatchHandler) HandleSBOMFilteredEvents(sfEvents <-chan watch.Event, pr
 			continue
 		}
 
+		if !slices.Contains(wh.managedInstanceIDSlugs, hashedInstanceID) {
+			wh.storageClient.SpdxV1beta1().OpenVulnerabilityExchangeContainers(obj.ObjectMeta.Namespace).Delete(context.TODO(), obj.ObjectMeta.Name, v1.DeleteOptions{})
```

Comment:
Run this flow only when the VEX capability is enabled

Reply:
If VEX capability is not enabled then there are no VEX objects to delete. I don't want to pollute the code by adding `if`-s all around these components. The nice thing that the whole system is object driven, if the object not there nothing happens